docker/build-packages/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -x
+
+# build jar
+# build web UI
+# build Debian and RPM packages
+# copy built packages into a mounted volume
+mvn clean package -Pbuild-ui \
+    && make all \
+    && cp *.deb *.rpm target/*.jar ${WORKDIR}/packages
```

Comment:
Could you rather do the following to avoid copying the original-cassandra-reaper-***.jar ? 

&& cp *.deb *.rpm target/cassandra*.jar ${WORKDIR}/packages